dtbase/backend/api/sensor/routes.py

```diff
@@ -202,7 +202,8 @@ def insert_sensor_readings():
 
 @blueprint.route("/list", methods=["GET"])
 # @login_required
-def list_sensors(type_name):
+# def list_sensors(type_name):
```

Comment:
I think we can just remove the commented-out line - it was a bug, as you pointed out.    Also, in the docstring, I think we can remove the word "of".